test/arbitrage.test.js

```diff
@@ -91,14 +105,36 @@ test('mapply 2 args with recycling', t => {
   t.true(r.all(r.is_equal(act,exp)))
 })
 
-test.todo('mapply 3 args')
-
+test('mapply 3 args', t => {
+  var act = r.mapply([1,2,3], [1,1,1], [2,2,2], (a, b, c) => a + b + c)
```

Comment:
These vectors violate the principle of no ambiguous results. There are multiple ways `mapply` could produce these results. e.g. it could reverse `[1,1,1]`, producing a correct result for this data but not for a different vector.